### DISCUSSION TITLE
message '; s.parentNode.insertBefore(g,s); })(); is shown on the log in page and edit page

### DISCUSSION THREAD

**Discussion by Sanja76** [requarks/wiki#3661] [Error / Bug Report]
**Describe the bug**
message '; s.parentNode.insertBefore(g,s); })(); appears on the log in page and the edit page of an article.

**To Reproduce**

1. Go to the log in page of the Wiki and observe the message in the top left hand corner of the page.
2. Log in to Wiki and select an article to Edit.
3. Click on the Edit icon.
4. Observe that when the Edit page is loading there is a message in the top left hand corner of the page '; s.parentNode.insertBefore(g,s); })();

**Expected behavior**
Message '; s.parentNode.insertBefore(g,s); })(); should not appear on log in or on edit article. 

**Screenshots**
![image](https://user-images.githubusercontent.com/63653878/79234214-3fc91f80-7e6a-11ea-84cd-871fdeef4b95.png)

**Host Info (please complete the following information):**
 - OS:Docker Container (Linux)
 - Wiki.js version: 2.2.51
 - Database engine: PostgreSQL 12.2

**Comment by Smankusors** [comment]
What browser do you use? Have you tried incognito mode?

**Comment by Sanja76** [comment]
I use Google Chrome Version 81.0.4044.92 (Official Build) (64-bit). I have tried with FireFox and Incognito mode but still the same message appears.

**Comment by Smankusors** [comment]
Are you open this from internal network or internet? Do you access it using HTTP or HTTPS?

**Comment by Sanja76** [comment]
I access it with HTTPS

**Comment by Smankusors** [comment]
I can't reproduce this, can you use fresh profile of your browser? Without any extensions?

**Comment by Smankusors** [comment]
Is this still happens with Wiki.js 2.4.x?
